### DISCUSSION TITLE
Which software you use for images in the book?

### DISCUSSION THREAD

**Discussion by shmuhammadd** [mlr-org/mlr3book#425] [General]
Hi, 

I am reading the book and I find out your graphics is awesome. Please, can you let me know which tool u use?

**Comment by pat-s** [comment]
Thanks! Which graphics?

**Comment by shmuhammadd** [comment]
Hi, this one : <img width="738" alt="image" src="https://user-images.githubusercontent.com/38854463/195983434-d2c04ba0-fa7d-4212-aff8-331c8ac979f0.png">

**Reply by pat-s** [reply]
@mb706

**Comment by mb706** [comment]
Most mlr3pipelines-graphics that look like this (rounded rectangles representing pipeops) were created with [google slides](https://www.google.com/slides/about/). There are a few [like this one](https://mlr3book.mlr-org.com/images/learner.svg) that were created using latex / tikz.
